### ISSUE TITLE
The Conjugate Gradient Is A Lie

### ISSUE THREAD

**Issue by jsuarez5341** [pytorch/pytorch#1359]
The torch.nn.optim docs promised me conjugate gradient descent (mentions it in the closure section), but it's not there. No references to the algorithm in the code base either :(. Not yet ported from Lua?

**Comment by ankitp94** [comment]
+1

**Comment by Breakend** [comment]
+1

**Comment by tttor** [comment]
+1

**Comment by samuelstanton** [comment]
+1

**Comment by BlindElephants** [comment]
+ 1

**Comment by williamFalcon** [comment]
+1

**Comment by sbarratt** [comment]
I implemented conjugate gradient in pytorch: https://github.com/sbarratt/torch_cg

**Comment by colesbury** [comment]
Maybe @vincentqb can chime in on whether we plan to include a conjugate gradient implementation

**Comment by vincentqb** [comment]
We would welcome a PR for it :)
